### PR TITLE
Adds civil war loadouts

### PR DESCRIPTION
**Pull request by blackdav123**
## About The Pull Request
Adds a redcoat and bluecoat loadout.
Adds full martini belts to make this work.

Adds quick equip vendors for both loadouts
## Why It's Good For The Game
Allows admins to easily run a civil war event by using combat patrol or sensor capture and replacing the vendors with civil war vendors.
## Changelog
:cl:
add: Added redcoat and bluecoat loadouts.
add: Added a full martini belt.
add: Added civil war quick equip vendors.
/:cl:

### CODE REVIEW

File: code/datums/quick_load_outfits.dm [hunk 1]
```diff
@@ -5,6 +5,8 @@
 	var/quantity = -1
 	///What job this loadout is associated with. Used for tabs and access.
 	var/jobtype = "Squad Marine"
+	///Restricts loadouts to a specific job. Set to false to allow any job to take the loadout.
+	var/requirejob = TRUE
```
Comment (Lumipharon) [inline code/datums/quick_load_outfits.dm:9]: camel_case

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Was this intended to be actually merged, or was this just a america day thing?

**Comment by blackdav123** [comment]
> Was this intended to be actually merged, or was this just a america day thing?

merged please, ive seen this event flop a few times due to being too hard to set up

**Comment by lbnesquik** [comment]
Aren't redcoats and bluecoats from the american revolt with the UK?

**Review by Lumipharon — APPROVED** [review]

File: code/datums/quick_load_civilwar.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/*!
+ * Any loadout that is intended for civil war vendors.
+ */
+
+///When making new loadouts, remember to also add the typepath to the list under init_civilwar_loadouts() or else it won't show up in the vendor
+
+/datum/outfit/quick/civilwar
```
Comment (Lumipharon) [inline code/datums/quick_load_civilwar.dm:7]: actually, more camel case

File: code/datums/quick_load_civilwar.dm [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+/datum/outfit/quick/civilwar/bluecoat
+	name = "Bluecoat"
+	jobtype = "Bluecoat"
+	desc = "Tally Ho! Show those redcoats a piece of your independence!"
+
+	w_uniform = /obj/item/clothing/under/marine/striped
+	mask = /obj/item/clothing/mask/bandanna/delta
+
+/datum/outfit/quick/civilwar/redcoat
```
Comment (Lumipharon) [inline code/datums/quick_load_civilwar.dm:36]: same

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Comment by Woker2004** [comment]
HOLY BASED, now we need a 13 second reload musket

**Pull request merged**